test/resources/layout-shared.css

```diff
@@ -45,3 +50,21 @@ body {
 .menu-playground {
     list-style: none;
 }
+
+.overlay-playground {
+    width: 400px;
+    padding: 50px 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
```

Comment:
What do you think about nesting the "modifiers" here?

I could be convinced either way, but I'm getting nervous we'll confuse our future selves mixing "real" and "fake" classes together when scanning down the classes in this file.

```
.overlay-playground {
  ...

  .fd-overlay--modal & {
    background: white;
  }
}
```

Reply:
I can't do nesting in this file because it's a CSS file.  We _could_ refactor it into sass, but that seemed like overkill.